telegram/_message.py

```diff
@@ -103,6 +103,8 @@ class Message(TelegramObject):
         from_user (:class:`telegram.User`, optional): Sender of the message; empty for messages
             sent to channels. For backward compatibility, this will contain a fake sender user in
             non-channel chats, if the message was sent on behalf of a chat.
+
+            .. seealso:: :any:`Conversationbot Example <examples.conversationbot>`
```

Comment:
the `Message` class & various of its attributes are used in all examples I guess. IMO there is no need to link to specific examples in this class

Reply:
deleted links to Conversationbot from Message class and its attrs/methods